docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md

```diff
@@ -94,10 +105,55 @@ This method retrieves a paginated list of all query logs.
 
     api_token = "Issued_API_TOKEN"
     base_url = "https://{your-engine-url}/api/v1/query/log"
+    search = "Search keyword(s)"
+    offset = {Offset}
+    limit = {Limit}
+
+    api_url = f"{base_url}?search={search}&offset={offset}&limit={limit}"
+
+    header = {
+        "Authorization": f"Bearer {api_token}"
+    }
+
+    r = requests.get(api_url, headers=header):
+    r.raise_for_status()
+    r.json()
+    ```
+
+=== "cURL"
+
+    ```shell
+      curl -X 'GET' \
+      'https://{your-engine-url}/api/v1/query/log/?search={search}&offset={offset}&limit={limit}' \
+      -H 'accept: application/json' \
+      -H 'Authorization: Bearer Issued_API_TOKEN'
+    ```
+
+### __Parameters__
+
+- `search`: Word(s) that the returned query logs should contain.
```

Comment:
```suggestion
- `search`: Word(s) that the query in the query log should contain.
```
We are searching for words in the query part of the query log so change the description to something in the same grain as above